src/content/resources/concepts/layer/implementation/component-definition/_index.md

```diff
@@ -9,6 +9,7 @@ aliases:
   - /docs/model/component/
   - /documentation/schema/component/
   - /documentation/schema/implementation-layer/component/
+    - /concepts/layer/implementation/component/
```

Comment:
BLOCKING: This is the line that caused my server error. **FIX:** remove leading space

Reply:
Nice catch, @Arminta-Jenkins-NIST, I will adjust that.

Reply:
Fixed in https://github.com/aj-stein-nist/OSCAL-Pages/commit/34e6dbb974e1a1af3b9ec3b8ad5d6b2569bde9fa.